config/rbac/role/role.yaml

```diff
@@ -603,6 +603,24 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - konnect.konghq.com
+  resources:
+  - konnectapiauthconfigurations
+  verbs:
+  - get
+  - list
+  - patch
+  - update
```

Comment:
Do we need the update permission for this resource?

Reply:
In order for the reconciler to be able to work with KonnectAPIAuthConfiguration resources, yes.